### DISCUSSION TITLE
1-Wire Temperaturfühler DS18B20

### DISCUSSION THREAD

**Discussion by Tailer67** [tbnobody/OpenDTU#1162] [Ideas]
Hallo,

ich fände eine Unterstützung für 1-Wire Temperarturfühler eine sinnvolle Ergänzung.
Zur Zeit erfasse ich mehrere Temperaturen z.B. der PV-Panel, Außentemperatur etc. separat mit einem ESP8266 der mit den 5 Volt (Onboard-Netzteil) der OpenDTU gespeist wird.

Bei einer 1-Wire Unterstützung könnte ich den zusätzlichen ESP8266 einsparen.

Grüße
Thomas

**Comment by Omega13x** [comment]
Der nächste möchte eine Steuerung fürs Aquarium, der Übernächste eine Fernbedienung für die Dusche, usw .........
Selber machen, oder mit dem zufrieden sein was geht. Zudem benötigt so ein 8266 nur ein paar Milliwatt mehr.
